docs/UsersGuide.asciidoc

```diff
@@ -565,13 +567,22 @@ via the blue Plus icon at the top of the table.
 The `Edit YAML` button can be used to reveal the YAML editor and migrate a group. After
 saving for the first time, the group can only be configured in YAML. The table view will not
 be shown anymore.
+Once a job group is configured in YAML, the test suites, products and machines used by it
+should not be renamed or deleted since that would create an inconsistent state. Instead the YAML
+should be updated to remove or replace the affected parts first.
```

Comment:
I would only mention this in the next section because it is a post-migration thing. (So I'd merge it with the similar note in the next section.)